### DISCUSSION TITLE
User Stories

### DISCUSSION THREAD

**Discussion by Ssunoo2** [ICS-414-In5PIRE/in5PIRE.github.io#5] [Announcements]
This is a place where we can write [user stories](https://courses.ics.hawaii.edu/ics414f24/modules/module-requirements-elicitation/) for more information

**Comment by Ssunoo2** [comment]
### CFO of Nonprofit Organization:
I am the chief financial officer of a nonprofit organization. As an autonomous organization, we make financial decisions from within and don't contract out to external accounting firms. I want a dashboard tool to provide financial insight on my organization's finances so that I can better allocate our resources and manage financial risks effectively.

**Comment by Ssunoo2** [comment]
### Small business owner
As a small business owner, I want to receive visually tangible graphs relating to financial solvency so that I may continue to run my business in the future given budgets and real time cash flow. This should allow me to proactively address any financial problems before any issues escalate.

**Comment by Ssunoo2** [comment]
### Analyst
I am an analyst and my projections are not always clear to non-technical coworkers. I want a tool that allows me to make projections but make them digestible to someone who may be technologically illiterate.

**Comment by bfd2** [comment]
### Financial Advisor

One of the best parts of this tool that I use often is the scenarios tool. Using the same data, I can create different scenarios and adjust it to fit into various situations so that we can predict and forecast accordingly in order to account for risk and differing volatility.

**Comment by bfd2** [comment]
### Certified Public Accountant

As a CPA, one of the biggest challenges is creating easy-to-understand data and information so that the client can easily understand what is happening and identifying significant trends in their finances. Having a template spreadsheet allows us to have a consistent sheet and takes out the hassle of learning what each different client’s needs are, allowing us to assist quicker and make it more manageable.

**Comment by bfd2** [comment]
### Auditor

As an auditor, analyzing and managing multiple years worth of financial data can be hard to skim through to make sure it is accurate and compliant with the law. Having the option to compare different years of data in easy to understand graphs and spreadsheets allow me to analyze data with ease.

**Comment by skulibas** [comment]
### HR Manager

As an HR manager, I need to track financial obligations related to employee benefits, such as Accrued Vacation and Workers' Compensation liabilities, categorized by due dates (within one year or after one year) so that I can ensure accurate planning for benefits-related payouts and budgeting.

**Comment by skulibas** [comment]
### Data Entry Specialist

As a data entry specialist, I need to upload pre-filled Excel sheets directly into input pages so that I can quickly populate financial data fields without entering each value manually. This feature saves time and reduces errors, as long as the Excel sheet follows the required format, ensuring data consistency.

**Comment by skulibas** [comment]
### Account Manager

As an account manager overseeing finances for multiple organizations, I need the ability to create and manage separate profiles for each organization under a single account so that I can keep their financial data organized and easily accessible. Each profile should have its own set of input sheets tailored to that organization, allowing me to quickly switch between profiles without mixing data.
